### PR TITLE
Cypress

### PR DESCRIPTION
**Pull request by anxolin**
Adds cypress integration

- Adds types
- Adds config + app id code
- Basic structure
- dummy test

It adds also 2 scripts, `test:integration` is meant to be used in a CI (travis). It will use a different port to launch the app `8081` so we can run it locally too with independence of having the app up or not.

For local, we can just run the app, as usual, and the do a `yarn cypress`

## Test it
```bash
# run it
yarn start
yarn cypress

# test CI
APP_ID=1 yarn build
yarn test:integration
```

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:8080",
+  "projectId": "digs58"
```
Comment (Velenir) [inline cypress.json:3]: randomly generated?

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr1584--dexreact.review.gnosisdev.com)**: Testing web app
   * **📚 [Storybook](https://pr1584--dexreact.review.gnosisdev.com/storybook/)**: Component stories

**Review by Velenir — COMMENTED** [review]
Did `cypress` bootstrap all these files for you or you had to copy-paste?

File: cypress.json (hunk 1 above)
Comment (anxolin) [reply cypress.json:3]: They assigned me that, I just created the project. We don't need it to run it, but just nice to have

Comment (anxolin) [reply cypress.json:3]: Not sure if u can see this https://dashboard.cypress.io/projects/digs58

**Review by Velenir — APPROVED** [review]

**Pull request merged**